dkron/utils.py

```diff
@@ -299,26 +298,18 @@ def __run_async_dkron(_command, *args, **kwargs) -> tuple[str, str]:
 
     name = f'tmp_{_command}_{time.time():.0f}'
 
-    if dkron_binary_version() >= (3, 2, 2):
-        # runoncreate was turned into asynchronous in https://github.com/distribworks/dkron/pull/1269
-        schedule = '@manually'
-        params = {'runoncreate': 'true'}
-    else:
-        schedule = f'@at {(timezone.now() + timezone.timedelta(seconds=5)).isoformat()}'
-        params = {}
-
     r = _post(
         'jobs',
         json={
             'name': add_namespace(name),
-            'schedule': schedule,
+            'schedule': '@manually',
```

Comment:
Why would hardcoding the schedule be required for the later versions?

Reply:
it has `runoncreate`, check the code that was removed above